src/features/dashboard/components/Dialogs/DeleteAppDialog/__tests__/delete-app-dialog.test.tsx

```diff
@@ -46,13 +46,13 @@ describe('Delete App Dialog', () => {
 
   it('Should render proper buttons', () => {
     const okButton = screen.getByRole('button', { name: /yes, delete/i });
-    const cancelButton = screen.getByRole('button', { name: /no, keep it/i });
+    const cancelButton = screen.getByRole('button', { name: /Cancel/i });
     expect(okButton).toBeInTheDocument();
     expect(cancelButton).toBeInTheDocument();
   });
 
   it('Should call onClose on cancel button click', async () => {
-    const cancelButton = screen.getByRole('button', { name: /no, keep it/i });
+    const cancelButton = screen.getByRole('button', { name: /cancel/i });
```

Comment:
[Question]: why one of the word starts with a capital letter and not the other one?

Reply:
Anything you write in between the ` //` it only checks for the words inside the ui and ignores capital letters. so their will be no differnce if we use capital or small letters in the test case